app/models/refine/filters/blueprint_editor.rb

```diff
@@ -59,6 +59,14 @@ def update(index, criterion:)
     blueprint[index][:input] = input
   end
 
+  def change_conjunction(index, conjunction_word)
+    if conjunction_word == "and"
+      blueprint[index][:word] = "and"
+    elsif conjunction_word == "or"
```

Comment:
No real use case for this behavior but I went ahead and added it since it was easy